bezier/src/main/java/io/channel/bezier/compose/component/checkbox/BezierCheckbox.kt

```diff
@@ -0,0 +1,293 @@
+import io.channel.bezier.BezierTheme
+import io.channel.bezier.compose.component.checkbox.properties.BezierCheckboxColor
+import io.channel.bezier.compose.component.checkbox.properties.BezierCheckboxStatus
+import io.channel.bezier.compose.component.checkbox.properties.BezierCheckboxVariant
+import io.channel.bezier.extension.thenIf
+import io.channel.bezier.icon.CheckBold
+import io.channel.bezier.icon.HyphenBold
+
+@Composable
+fun BezierCheckbox(
+        color: BezierCheckboxColor,
+        variant: BezierCheckboxVariant,
+        status: BezierCheckboxStatus,
+        onClick: (BezierCheckboxStatus) -> Unit,
+        modifier: Modifier = Modifier,
+        text: String? = null,
+        enabled: Boolean = true,
+) {
+    val colorSchemes = color.getColorSchemes(variant, status)
+    val backgroundColor = colorSchemes.backgroundColor().color
+    val iconColor = colorSchemes.iconColor().color
+
+    Row(
+            modifier = modifier
+                    .thenIf(!enabled) {
+                        alpha(0.4f)
+                    }
+                    .clickable(enabled = enabled) {
+                        onClick.invoke(status.switch())
+                    },
+    ) {
+        if (variant == BezierCheckboxVariant.Secondary) {
+            Spacer(modifier = Modifier.width(32.dp))
+        }
+
+        BezierCheckboxControl(
+                modifier = Modifier.padding(variant.controlPadding),
+                backgroundColor = backgroundColor,
+                iconColor = iconColor,
+                variant = variant,
+                status = status,
+                onClick = {
+                    onClick.invoke(status.switch())
+                },
+                enabled = enabled,
+        )
+
+        if (text != null) {
+            BezierCheckboxLabel(
+                    text = text,
+                    modifier = modifier.align(Alignment.CenterVertically),
+            )
+        }
+    }
+}
+
+@Composable
+private fun BezierCheckboxControl(
+        status: BezierCheckboxStatus,
+        variant: BezierCheckboxVariant,
+        backgroundColor: Color,
+        iconColor: Color,
+        enabled: Boolean,
+        onClick: () -> Unit,
+        modifier: Modifier = Modifier,
+) {
+    Box(
+            modifier = modifier
+                    .size(24.dp)
+                    .padding(2.dp)
+                    .clip(CircleShape)
+                    .clickable(
+                            enabled = enabled,
+                            onClick = onClick,
+                    )
+                    .background(backgroundColor),
+            contentAlignment = Alignment.Center,
+    ) {
+        if (enabled && variant == BezierCheckboxVariant.Primary && status == BezierCheckboxStatus.False) {
+            Box(
+                    modifier = Modifier
+                            .size(16.dp)
+                            .clip(CircleShape)
+                            .background(iconColor),
```

Comment:
```suggestion
                            .background(BgWhiteNormal),
```
여기는 아이콘 컬러랑 다르네요!
<img width="220" alt="image" src="https://github.com/user-attachments/assets/ebbd7683-fa4e-4b3b-bba3-75c13c96ceaa">

Reply:
이거 아닌가요!?
<img width="723" alt="image" src="https://github.com/user-attachments/assets/71300c89-34d8-4440-8a89-3e0a3ec7b76b">

Reply:
디자인 상으로는 이 부분을 아이콘으로 보는게 아니라 `흰 배경 + 테두리` 로 보기 때문에 bg 컬러가 들어간것 아닐까 싶네용
사소한것 같아서 잘 모르겠긴 한데 아래 방법 중 하나로 해두면 좋을것 같네욥
- 피그마랑 동일하게 맞추기
- 주석 달아두기
- 컴포넌트 피드백하기
- 그냥 두기
<img width="993" alt="image" src="https://github.com/user-attachments/assets/a5eb1c8a-ecd2-4c1d-8858-acb91c04a23e">

Reply:
착각했네욥..!